docs/modules/ROOT/examples/map.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: hazelcast.com/v1alpha1
+kind: Hazelcast
+metadata:
+  name: map-sample
+spec:
+  hazelcastResourceName: hazelcast
```

Comment:
I think we should update `hazelcastResourceName` like you did for the map name(map-sample)

Reply:
I agree I will do it
```suggestion
  hazelcastResourceName: hazelcast-sample
```